packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -1068,6 +1081,52 @@ class ApiGateway {
     }
   }
 
+  public async sqlRunner({ query, context, res }: QueryRequest) {
+    const requestStarted = new Date();
+    try {
+      if (!query) {
+        throw new UserError('query param is required');
```

Comment:
Could you please change the text of the errors? We are trying to use meaningful sentences. Ex: 
`throw new UserError('A user\'s selector must contain at least one time zone.');`